lib/retrospective-assessment.js

```diff
@@ -0,0 +1,71 @@
+const {
+  some,
+  intersection,
+  values,
+  flatten,
+  isUndefined,
+  isPlainObject
+} = require('lodash');
+const { projectSpecies } = require('@asl/constants');
+
+const species = flatten(values(projectSpecies));
+
+const nopes = [
+  // legacy
+  'prosimians',
+  'marmosets',
+  'cynomolgus',
+  'rhesus',
+  // legacy
+  'vervets',
+  // legacy
+  'baboons',
+  // legacy
+  'squirrel-monkeys',
+  // legacy
+  'other-old-world',
+  // legacy
+  'other-new-world',
+  'other-nhps',
+  // legacy
```

Comment:
It's a little bit unclear as to which of the species the legacy comment refers to - I assume it's just the immediate line below? Might be clearer to move them inline, e.g. ```'apes', // legacy```